lego/apps/users/serializers/users.py

```diff
@@ -5,14 +5,15 @@
 from lego.apps.users import constants
 from lego.apps.users.models import AbakusGroup, Penalty, User
 from lego.apps.users.serializers.abakus_groups import PublicAbakusGroupSerializer
-from lego.apps.users.serializers.memberships import MembershipSerializer
+from lego.apps.users.serializers.memberships import MembershipSerializer, PastMembershipSerializer
 from lego.apps.users.serializers.penalties import PenaltySerializer
 from lego.utils.fields import PrimaryKeyRelatedFieldNoPKOpt
 
 
 class DetailedUserSerializer(serializers.ModelSerializer):
 
     abakus_groups = PublicAbakusGroupSerializer(many=True)
+    past_memberships = PastMembershipSerializer(many=True)
```

Comment:
This only adds two queries (only 1 with `.select_related('abakus_group')`), since we only use these serializers on detail routes, atleast that is what I have found.